google/cloud/aiplatform/training_jobs.py

```diff
@@ -1445,6 +1480,18 @@ def _wait_callback(self):
 
             self._has_logged_custom_job = True
 
+        if (
+            self._gca_resource.training_task_metadata
+            and self._gca_resource.training_task_metadata.get("backingCustomJob")
+            and self._gca_resource.training_task_inputs.get("enable_web_access")
+            and not self._has_logged_web_access_uris
```

Comment:
Is it possible the `web_access_uris` have changed throughout the run? If, for example, one of the workers failed.

Reply:
The current service setup is that if workers failed and restarted, the `web_access_uris` will redirect to the new workers, but won't change itself.